src/containers/ BooksForm.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+export const BOOKCATES = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const BooksForm = ({ addBook }) => {
+  const [formData, setFormData] = useState({ title: '', category: '' });
+
+  const handleChange = (e) => {
+    setFormData((formData) => ({ ...formData, [e.target.name]: e.target.value }));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    addBook(formData);
+    setFormData({ title: '', category: '' });
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input
+        id="title"
+        type="text"
+        name="title"
+        placeholder="Title"
+        onChange={handleChange}
+        required
+      />
+      <select>
+        <option>1</option>
+      </select>
```

Comment:
- [ ] The BookForm should have a select element that displays all categories. The form functionality should not be working, for now, clicking on the submit button should not trigger any action at the current milestone

Reply:
Done